sdk/monitor/azure-monitor-query/LargeLogQuery.md

```diff
@@ -0,0 +1,123 @@
+# Running Large Log Queries using Log Analytics
+
+This guide will show you how to run large log queries **without** extra costs using Log Analytics and the Monitor Query SDK.
+
+Due to Log Analytics [service limits](https://learn.microsoft.com/en-us/azure/azure-monitor/service-limits#la-query-api), sometimes it may
+not be possible to retrieve all the expected data in a single query. For example, the number of rows returned or the maximum size of the
+data returned may exceed the stated limits. One approach for overcoming these limits is to split the queries into multiple smaller queries
+using different time ranges.
```

Comment:
Queries can be split into smaller sub-queries based on time ranges only for certain types of queries - for e.g. aggregations may not work well.